src/main/java/org/jboss/threads/ViewExecutor.java

```diff
@@ -100,6 +105,14 @@ public Builder setQueueInitialSize(final int queueInitialSize) {
         }
 
         public ViewExecutor build() {
+            if (USE_V2) {
+                // queueInitialSize is not relevant using ConcurrentLinkedQueue
+                return new EnhancedViewExecutor(
+                        Assert.checkNotNullParam("delegate", delegate),
+                        maxSize,
+                        queueLimit,
+                        handler);
+            }
             if (queueLimit == 0) {
                 return new QueuelessViewExecutor(Assert.checkNotNullParam("delegate", delegate), maxSize, handler);
```

Comment:
The QueuelessViewExecutor can safely be deleted since it has not been released and the EnhancedViewExecutor performs just as well.

Depending on risk tolerance the QueuedViewExecutor and feature flag could be removed as well. I defer to your judgement.

Reply:
Let's definitely delete Queueless.  Less code means fewer bugs.

Reply:
Sounds good. Do you have any preference on the default value of the `org.jboss.threads.view-executor.v2` flag to use the enhanced version when a queue is requested?
If we leave it enabled by default, I'll add javadoc and a deprecation to the get/setQueueInitialSize methods.

Reply:
Let's just have one implementation; we don't need to keep the old one around IMO.  I think it's probably fine to deprecate those methods.

Reply:
Sounds reasonable. I've deleted the QueuedViewExecutor implementation.

It may be helpful to add padding to prevent noisy neighbor problems, wdyt? Perhaps not necessary for this PR.